packages/zwave-js/src/lib/driver/UpdateConfig.ts

```diff
@@ -64,11 +65,38 @@ export async function installConfigUpdate(newVersion: string): Promise<void> {
 		);
 	}
 
+	const pkgFilepath = `${pak.cwd}/package.json`;
+
+	await lockfile.lock(pkgFilepath, {
```

Comment:
This can throw if we cannot acquire the lock. In that case we should return a better (ZWave) error. The code can be `Config_Update_InstallFailed`.